src/Workspaces/Core/Portable/Options/EditorConfig/IEditorConfigOptionsCollection.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Options
+{
+    internal interface IEditorConfigOptionsCollection
+    {
+        public abstract IEnumerable<(string feature, ImmutableArray<IOption2> options)> GetLanguageSpecificEditorConfigOptions();
```

Comment:
>GetLanguageSpecificEditorConfigOptions [](http://example.com/codeflow?start=88&length=38)

Consider shortening the name to just `GetOptions` - the interface already specifies that these are editor config options. The interface itself isn't necessarily language specific. It's just used that way.